Workbooks/ADXCluster/AtResource/ResourceInsights.workbook

```diff
@@ -580,28 +2392,357 @@
+                    "formatter": 4,
+                    "formatOptions": {
+                      "min": 0,
+                      "palette": "red"
+                    },
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "useGrouping": false,
+                        "maximumFractionDigits": 2
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--cachemisses Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--errors",
+                    "formatter": 4,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "maximumFractionDigits": 1
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--errors Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfExtents Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfExtents",
+                    "formatter": 1,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": null
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfThrottledCommands Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfThrottledCommands",
+                    "formatter": 1,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": null
+                    }
+                  }
+                ],
+                "rowLimit": 10000,
+                "filter": true,
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "Subscription"
+                  ],
+                  "expandTopLevel": true,
+                  "finalBy": "Name"
+                },
+                "labelSettings": [
+                  {
+                    "columnId": "Subscription"
```

Comment:
we should also have a `"label":"Subscription"` here so our loc team can translate this. We should've automatically generated this on our end but we need to fix that. So for now we need to add the labels where they are missing.